runtime/doc/vim9class.txt

```diff
@@ -710,6 +710,32 @@ The initialization isn't needed, the list is empty by default.
 							*E1330*
 Some types cannot be used, such as "void", "null" and "v:none".
 
+Builtin Object Methods ~
+                                                       *builtin-object-methods*
+Some of the builtin functions like |empty()|, |len()| and |string()| can be
+used with an object.  An object can implement a method with the same name as
+these builtin functions to return an object-specific value.
+
+                                                       *E1412*
+The following builtin methods are supported:
+                                                       *object-empty()*
+    empty()  Invoked by the |empty()| function to check whether an object is
+	     empty.  If this method is missing, then true is returned.  This
```

Comment:
Not sure if this section should be reworked or removed in favor of `builtin-interfaces`. If reworked, maybe something like
```
    Empty()  Invoked by the |empty()| function, if the |vimContainer|
             interface is defined, to check whether an object is empty.  If
             the interface is missing, then the method is ignored and
             |empty()| returns true. This method should not accept any
             arguments and must return a boolean.
```

Reply:
It seems too wordy. At a minimum
> This method should not accept any
> arguments and must return a boolean.

is probably not needed, vimContainer, validates/requires a correct signature. And the signature is mentioned under vimContainer.

Reply:
Yes.  I forgot to remove the `builtin-object-methods` section from the help.  I have updated the PR with this change.  Thanks.